File: 2061.py
qtd_abas, qtd_acoes = map(int, input().split())

for i in range(qtd_acoes):
	acao = input()

	if acao == "fechou":
		qtd_abas -= 1
		qtd_abas += 2

	else:
		qtd_abas -= 1

print(qtd_abas)

File: 2061.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
  int n, m;
  string acao;

  cin >> n >> m;

  for (int i = 0; i < m; ++i) {
    cin >> acao;

    if (acao == "fechou") {
      n -= 1;
      n += 2;
    } else {
      n -= 1;
    }
  }

  cout << n << endl;

  return 0;
}